src/components/geojs/GeojsGeojsonLayer.vue

```diff
@@ -0,0 +1,50 @@
+<template lang="pug">
+// geojs geojson layer
+</template>
+
+<script>
+import bindWatchers from './bindWatchers';
+import { layerMixin } from './mixins';
+
+export default {
+  mixins: [layerMixin],
+  props: {
+    geojson: {
+      type: [Object, Array],
```

Comment:
Looks like this prevent passing `null`. I was think passing `null` for an empty state.

Reply:
I was thinking just `{}` for null, but actually there are other types that can be passed into the json reader as well (strings, blobs, etc.).  I just disabled type checking for the property.